src/ninshiki_cpp/detector/dnn_detector.cpp

```diff
@@ -38,15 +38,20 @@ DnnDetector::DnnDetector(bool gpu, bool myriad)
   model_suffix = utils::split_string(model, ".");
   net = cv::dnn::readNet(model, config, "");
 
-  this->gpu = gpu;
-  this->myriad = myriad;
-
   std::ifstream ifs(file_name.c_str());
   std::string line;
   while (std::getline(ifs, line)) {
     classes.push_back(line);
   }
 
+
+}
+
+void DnnDetector::set_computation_method(bool gpu, bool myriad)
+{
+  this->gpu = gpu;
+  this->myriad = myriad;
```

Comment:
Don't change this in here, but having an enum is better than using these variables.